src/components/App.tsx

```diff
@@ -20,7 +21,12 @@ export function App(): ReactElement {
 
   useEffect(() => {
     let aborted = false;
-    void getVersion().then((version) => !aborted && setVersion(version));
+    getVersion().then((version) => {
+      if (!aborted) {
+        setWrapperVersion(version['inferno-framework/fhir-validator-wrapper']);
+        setHl7ValidatorVersion(version['org.hl7.fhir.validation']);
```

Comment:
Marking this as void fixes the error about awaiting promises, but these lines will see other type errors